### DISCUSSION TITLE
Interested in font

### DISCUSSION THREAD

**Discussion by PrtmPhlp** [poetaman/arttime#14] [General]
Hey, first of all, I love your little program. It looks clean, nice and is really customizable. 
Now to the question, in the photos of your Readme, the font and your program fits together very well. Do you know, what font that was?

**Comment by poetaman** [comment]
@PrtmPhlp Moving this to discussions. Am glad you are liking arttime :)  It's Operator Mono. If you are looking for a similar but free font, then search the web for a monospace font with cursive italics. I think I had seen something similar and free, though not exactly the same...

**Reply by PrtmPhlp** [reply]
Thanks for the reply! Found a font called "Victor mono", which looks really similar. Thanks again for the help

**Comment by poetaman** [comment]
Update: Most screenshots on front page -- ones with cursive italics -- use Operator Mono. I have moved on (mostly) to another font that is visible in new screenshots -- one without cursive italics -- its name is Pragmata Pro. Victor Mono seems like a similar and free alternative to Operator Mono. And Pragmasevka is a free alternative to Pragmata Pro.

**Reply by PrtmPhlp** [reply]
Thanks for the update! 😃
